packages/flutter/test/rendering/platform_view_test.dart

```diff
@@ -98,15 +100,64 @@ void main() {
     });
 
   }, skip: isBrowser); // TODO(yjbanov): fails on Web with obscured stack trace: https://github.com/flutter/flutter/issues/42770
+
+  // Regression test for https://github.com/flutter/flutter/issues/69431
```

Comment:
did you confirm that reverting the changes in platform_views.dart causes this test to fail?

Reply:
Yes,this can reproduce the issue.